pageserver/src/tenant.rs

```diff
@@ -4482,13 +4482,17 @@ impl Tenant {
         let mut gc_cutoffs: HashMap<TimelineId, GcCutoffs> =
             HashMap::with_capacity(timelines.len());
 
+        // Ensures all timelines use the same start time when computing the time cutoff.
+        let now = SystemTime::now();
```

Comment:
I think a more descriptive name would be better like `ts_for_pitr_calc`. Then people will also think more before they touch that variable.